src/test/java/org/ays/tests/institution/adminregistrationmanagementservice/PostAdminRegistrationApplicationTest.java

```diff
@@ -36,7 +36,7 @@ public void createAnAdminRegistrationApplication() {
                 .body("response", hasKey("id"));
     }
 
-    @Test(dataProvider = "invalidDataForPostApplicationReasonField")
+    @Test(dataProvider = "invalidDataForPostApplicationReasonField",dataProviderClass = DataProvider.class)
```

Comment:
Kod formatında problem var. Bu tür durumlar yaşamamak için aşağıdaki dokümanı inceleyebilirsiniz;
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html